modules/serviceregistry/src/main/java/org/opencastproject/serviceregistry/impl/endpoint/ServiceRegistryEndpoint.java

```diff
@@ -687,6 +687,20 @@ public Response getMaxLoadOnNode(@QueryParam("host") String host) throws NotFoun
     }
   }
 
+  @GET
+  @Path("maxhardwareload")
+  @Produces(MediaType.TEXT_PLAIN)
```

Comment:
Shouldn't it be better to return JSON?

Reply:
I adapted this changes:

`
-  @Produces(MediaType.TEXT_PLAIN)
+  @Produces(MediaType.APPLICATION_JSON)
   @RestQuery(name = "maxhardwareload", description = "Returns the maximum hardware load this service registry can execute concurrently.",
       returnDescription = "The  maximum hardware load this service registry can execute", restParameters = {},
       responses = { @RestResponse(responseCode = SC_OK, description = "Maximum hardware load this service registry can execute.") })
   public Response getMaxHardwareLoad() {
     try {
-      return Response.ok(serviceRegistry.getMaxHardwareLoad()).build();
+      JSONArray list = new JSONArray();
+
+      list.add(serviceRegistry.getMaxHardwareLoad());
+
+      return Response.ok(list.toString()).build();
`